server/forge/common/status.go

```diff
@@ -38,12 +38,13 @@ func GetPipelineStatusContext(repo *model.Repo, pipeline *model.Pipeline, workfl
 	}
 	var ctx bytes.Buffer
 	err = tmpl.Execute(&ctx, map[string]any{
-		"context":  server.Config.Server.StatusContext,
-		"event":    event,
-		"workflow": workflow.Name,
-		"owner":    repo.Owner,
-		"repo":     repo.Name,
-		"axis_id":  workflow.AxisID,
+		"context":        server.Config.Server.StatusContext,
+		"event":          event,
+		"workflow":       workflow.Name,
+		"workflow_count": len(pipeline.Workflows),
```

Comment:
this is the parsed pipeline ... we must make sure that even if workflows are filtered out later, they still count

Reply:
If this is the parsed pipeline and if workflows are filtered out _later_, then they still count here, no?

Reply:
i dont remember that's what to find out ...

Reply:
Is there a simple solution to get that info?

Reply:
I think it's not worth it